javascript/javascript-maps.md

```diff
@@ -0,0 +1,122 @@
+# Javascript Maps
+
+## Projected Time
+
+30-45 minutes
+
+## Prerequisites
+
+- [Javascript 6 Lesson](https://github.com/Techtonica/curriculum/blob/master/javascript/javascript-6-object-literals.md)
```

Comment:
@imskr Could the link be made short like `../javascript/javascript-6-object-literals.md` ?